esp-hal/src/i2c.rs

```diff
@@ -112,10 +112,11 @@ pub enum Error {
 
 #[cfg(any(feature = "embedded-hal", feature = "async"))]
 #[derive(PartialEq)]
-// This enum is used to keep track of the last operation that was performed
-// in an embedded-hal(-async) I2C::transaction. It used to determine whether
-// a START condition should be issued at the start of the current operation.
-enum LastOpWas {
+// This enum is used to keep track of the last/next operation that was/will be
+// performed in an embedded-hal(-async) I2C::transaction. It used to determine
```

Comment:
You might consider "It is used" here?

I noticed this (presumably typo) some days back. Interestingly, also the current text is textually correct, but refers to something that was in the past.

Reply:
Fixed!